pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -121,6 +121,48 @@ static __always_inline bool kafka_process(kafka_transaction_t *kafka_transaction
 
     log_debug("kafka: topic name is %s", kafka_transaction->topic_name);
 
+    switch (kafka_header.api_key) {
+    case KAFKA_PRODUCE:
+    {
+        READ_BIG_ENDIAN_WRAPPER(s32, number_of_partitions, skb, offset);
+        if (number_of_partitions > 1) {
+            log_debug("We have more than a single partition for produce request, we currently support only the first partition");
```

Comment:
Either rephrase the log or the code, as the log says we support only the first partition, but the if clause above says we support messages with **a single** partition

so either we parse only the first partition (no matter how many partition exist) or the log should say we support messages with a single partition

Reply:
Good call, fixed